source/vibe/inet/url.d

```diff
@@ -39,7 +39,7 @@ struct URL {
 		m_schema = schema;
 		m_host = host;
 		m_port = port;
-		m_pathString = path.toString();
+		m_pathString = urlEncode(path.toString(), "/");
```

Comment:
This change the behaviour of everything else that is using this method. I agree the current behaviour is bug prone, but I'm not keen on doing that.